.pre-commit-hooks.yaml

```diff
@@ -318,4 +318,3 @@
   entry: 809245501444.dkr.ecr.eu-central-1.amazonaws.com/openapi-linter:release-latest-main
   language: docker_image
   pass_filenames: false
-  files: '^apidoc/**/{openapi,asyncapi}.yaml$'
```

Comment:
Looks like you want:

```
files: '^apidoc/[^/]+/{openapi,asyncapi}.yaml$'
```

It's a regex not a glob expression.

Reply:
Ideally you'd also have `pass_filenames: true` so it only runs on specific files that change, but idk how easy it is for docker-based pre-commit hooks.

Might not be worth the trouble now at least.

Reply:
should that be
```
files: '^apidoc/[^/]+/(openapi|asyncapi).yaml$'
```

Reply:
Yes what Samer said 😅

Reply:
Even better:

```
files: '^apidoc/[^/]+/(openapi|asyncapi)\\.yaml$'
```